### PR TITLE
Add docs index, PR template and fix typo in contribute

### PR DESCRIPTION
**Pull request by gtrkiller**
### Overview

Follow up to #24 to fix what is needed. Also adds the PR template for future PRs

### Rationale

N/A

### Juju Events Changes

N/A

### Module Changes

N/A

### Library Changes

N/A

### Checklist

- [X] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [X] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [X] The changes are compliant with [ISD054 - Manging Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [X] The documentation is generated using `src-docs`
- [X] The documentation for charmhub is updated.
- [X] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Comment by jdkandersson** [comment]
Is this missing the description defined in ISD062?

File: docs/index.md [hunk 1]
```diff
@@ -31,3 +31,19 @@ Pollen is an open-source project that welcomes community contributions, suggesti
 Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, or ask a question, or make a suggestion about a potential change via the comments section.
 
 If there's a particular area of documentation that you'd like to see that's missing, please [file a bug](https://github.com/canonical/pollen-operator/issues).
+
+# Navigation
```
Comment (jdkandersson) [inline docs/index.md:35]: Should this use the contents index format? https://github.com/canonical/upload-charm-docs#contents-index

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!--
+Thank you for your interest in and contributing to Synapse Operator!
```
Comment (amandahla) [inline .github/pull_request_template.md:2]: Synapse? ;-)

**Review by jdkandersson — CHANGES_REQUESTED** [review]
Please ensure this PR complies with ISD062

**Comment by merkata** [comment]
> Please ensure this PR complies with ISD062

@gtrkiller you can copy a template from [here](https://github.com/canonical/synapse-operator/pull/30) for instance and make use of it until the template is replicated across all our repos. Please note that if a checkbox is not applicable, you should check it and not leave it empty. Leaving it empty leaves the impression it is not done yet, marking it signals it is either done or not applicable for this PR. Thanks!

PS add a label ;)

**Review by amandahla — DISMISSED** [review]
LGTM

**Review by nrobinaubertin — APPROVED** [review]

**Review by jdkandersson — COMMENTED** [review]
There are still some earlier comments that were marked as resolved but don't seem to be resolved yet, I have un-resolved them, please take a look

**Review by jdkandersson — APPROVED** [review]
Marking as approved to remove the requested changes tag, please address the earlier comment

**Review by jdkandersson — APPROVED** [review]

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by arturo-seijas — APPROVED** [review]

**Pull request merged**